python/semantic_kernel/connectors/ai/open_ai/utils.py

```diff
@@ -167,7 +174,17 @@ async def chat_completion_with_function_call(
     # if there is no function_call or if the content is not a FunctionCall object, return the context
     if function_call is None or not isinstance(function_call, FunctionCall):
         return context
-    result = await execute_function_call(kernel, function_call, log=log)
+    if function_call_with_new_context:
```

Comment:
Are there any test cases that need to be added or updated related to this change?

Reply:
This utility func is currently not tested, I will have a look.

Reply:
We can always create a separate item to track adding tests.

Reply:
sounds good, will start that asap